Sources/XcodeProj/Objects/Files/PBXFileSystemSynchronizedRootGroup.swift

```diff
@@ -83,14 +83,14 @@ public class PBXFileSystemSynchronizedRootGroup: PBXFileElement {
 
     // MARK: - PlistSerializable
 
-    override var multiline: Bool { true }
+    override var multiline: Bool { (exceptions?.count ?? 0) < 2 }
```

Comment:
This condition is somewhat weird, but it looks like Xcode merges everything into a single line when there is more than one exception? It's definitely odd, though.